### PR TITLE
add loader for refresh volumes

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**: ui

**Context**: As a user, I would like to see the loader when the volume list is refreshing.

**Summary**: We add a loader above the volume list when the volume is refreshing.

**Acceptance criteria**: 
image

See: #1459

### CODE REVIEW

File: ui/src/ducks/app/volumes.js [hunk 1]
```diff
@@ -234,6 +275,23 @@ export function* fetchPersistentVolumeClaims() {
     yield put(setPersistentVolumeClaimAction(result.body.items));
   }
 }
+export function* refreshPersistentVolumes() {
```
Comment (carlito-scality) [inline ui/src/ducks/app/volumes.js:279]: LGTM but need unit test here.

Comment (ChengYanJin) [reply ui/src/ducks/app/volumes.js]: Thanks Carlito! I added unit test for refresh persistent volume.

Comment (ChengYanJin) [reply ui/src/ducks/app/volumes.js:279]: > LGTM but need unit test here.

Thanks Carlito! I added unit test for refresh persistent volume.

**Review by slaperche-scality — DISMISSED** [review]
Works as described

**Comment by ChengYanJin** [comment]
@bert-e approve

**Review by slaperche-scality — APPROVED** [review]

**Pull request merged**